.gitignore

```diff
@@ -1,4 +1,7 @@
-.DS_Store
+# NOTE: Do not ignore .DS_Store for test fixtures.
+# .DS_Store
+./.DS_Store
```

Comment:
Does this mean `.DS_Store` won't be ignored in other parts of the repo

Reply:
Yea it does for now.